corehq/util/test_utils.py

```diff
@@ -347,6 +348,12 @@ def get_output(self):
         return self.output.getvalue()
 
 
+def unregistered_django_model(model_class):
```

Comment:
Hey @millerdev 

Can we add a comment why this is needed?
Looks like this would be needed now every time we instantiate an abstract class, though that should not be done anyway right?
Alternatively, We can't update the tests to use a class that can be instantiated instead?

Reply:
Good idea. https://github.com/dimagi/commcare-hq/pull/31319/commits/2ac34f00b8c92fe74c00bf21e3988a8552bd7b75

Reply:
Thanks for the change @millerdev . Looks good.